test/fail_compilation/extra-files/mixin-postscript.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+rm -f "test.mixin"
```

Comment:
As you noticed this doesn't work.

Reply:
It works locally, but not for circleci !?

Reply:
Yes, because one shall not create temporary files in the git repository. CircleCi was checking this and the solution shouldn't have been to hack the check, but placing the file in `TEST_RESULTS`